buzzmobile/launch/lidar_test/lidar_test.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+    <rosparam command="load" file="$(find buzzmobile)/constants.yaml"/>
+	<include file="$(find buzzmobile)/launch/lidar_test/mock_sensors.launch"/>
+	<include file="$(find buzzmobile)/launch/includes/lidar.launch"/>
+	<include file="$(find buzzmobile)/launch/lidar_test/visualization.launch"/>
```

Comment:
Also this file is duplicated in both `buzzmobile/launch/lidar_test/lidar_test.launch` and `buzzmobile/launch/lidar_test.launch`.

If you make the dir structure as I said in a different comment this will be fixed.